jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -144,24 +153,68 @@ pipeline {
+                        def local_component = component.trim()
+                        def local_component_index = componentList.indexOf(local_component)
+                        def wait_seconds = local_component_index * 10
+
+                        echo "Add Component: ${local_component}"
+                        componentTests["Run Integtest ${local_component}"] = {
+                            // Using scripted pipelines to trigger dynamic parallel stages
+                            timeout(time: 2, unit: 'HOURS') {
+                                node(AGENT_LABEL) {
+                                    docker.withRegistry('https://public.ecr.aws/') {
+                                        docker.image(docker_images["$distribution"]).inside(docker_args["$distribution"]) {
+                                            try {
+                                                stage("Run Integtest ${local_component}") {
+                                                    echo "Component Name: ${local_component}"
+                                                    // Jenkins tend to not clean up workspace at times even though ws clean is called
+                                                    // Due to docker is mounting the agent directory so it can communicated with the agent
+                                                    // This sometimes causes the workspace to retain last run test-results and ends with build failures
+                                                    // https://github.com/opensearch-project/opensearch-build/blob/6ed1ce3c583233eae4fe1027969d778cfc7660f7/src/test_workflow/test_recorder/test_recorder.py#L99
+                                                    sh("echo ${local_component} with index ${local_component_index} will sleep ${wait_seconds} seconds to reduce load && sleep ${wait_seconds}")
+                                                    unstash "integtest-opensearch-dashboards-$BUILD_NUMBER"
+                                                    sh("rm -rf test-results")
+                                                    runIntegTestScript(
+                                                        jobName: "$BUILD_JOB_NAME",
+                                                        componentName: "${local_component}",
+                                                        buildManifest: "$BUILD_MANIFEST",
+                                                        testManifest: "manifests/${TEST_MANIFEST}",
+                                                        localPath: "${WORKSPACE}/${distribution}",
+                                                        switchUserNonRoot: "${switch_user_non_root}"
+                                                        )
+                                                }
+                                            } catch (e) {
+                                                echo "Error running integtest for component ${local_component}"
```

Comment:
Can you add creating GH issue as well? Similar to https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch/integ-test.jenkinsfile#L179-L185

Reply:
Can I keep the issue title same, as the Component name has `dashboards` in it @gaiksaya Everything seems to be similar.

Reply:
Yes! Should be okay!

Reply:
This feature will be added per https://github.com/opensearch-project/opensearch-build/issues/3476 once #3474 is fully built and synced to hub/ecr.

cc: @peterzhuamazon